chat-widget/src/components/livechatwidget/livechatwidgetstateful/LiveChatWidgetStateful.tsx

```diff
@@ -188,21 +189,23 @@ export const LiveChatWidgetStateful = (props: ILiveChatWidgetProps) => {
     }, [props.webChatContainerProps?.webChatStyles]);
     
     const webChatProps = initWebChatComposer(props, chatSDK, state, dispatch, setWebChatStyles);
-    const setPostChatContextRelay = () => setPostChatContextAndLoadSurvey(chatSDK, dispatch, true);
+    const setPostChatContextRelay = () => setPostChatContextAndLoadSurvey(chatSDK, dispatch);
     const endChatRelay = () => endChat(props, chatSDK, setAdapter, setWebChatStyles, dispatch, adapter);
     const prepareStartChatRelay = () => prepareStartChat(props, chatSDK, state, dispatch, setAdapter);
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    const initStartChatRelay = (optionalParams?: any) => initStartChat(chatSDK, dispatch, setAdapter, optionalParams);
+    const initStartChatRelay = (optionalParams?: any, persistedState?: any) => initStartChat(chatSDK, dispatch, setAdapter, optionalParams, persistedState);
```

Comment:
Why do we need `persistedState`? Isn't `optionalParams` enough to pass `liveChatContext` to `startChat()`?

Reply:
`persistedState` has the cached livechatcontext that we pass through in `optionalParams` while reconnecting to chat, this is used during multitab scenarios